### PR TITLE
RF 03.02 | Multi-Configuration >Enable \ Disable project

### PR DESCRIPTION
**Pull request by KiraKomissarova**
https://trello.com/c/J9lv7cia

### CODE REVIEW

**Review by Elygan — APPROVED** [review]

File: src/test/java/school/redrover/MultiConfigurationProject4Test.java [hunk 1]
```diff
@@ -35,6 +35,33 @@ public void testEnableMultiConfigurationProject() {
...
+
+        WebElement enableButton = new MultiConfigurationProjectPage(getDriver())
+                .getConfigPage()
+                .switchCheckboxEnabled()
+                .getTextEnabled();
+
+        Assert.assertEquals(enableButton.getText(), "Enabled");
+    }
+
     @DataProvider(name = "unsafeCharacter")
```
Comment (Maria7725) [inline src/test/java/school/redrover/MultiConfigurationProject4Test.java:65]: Kira, you've committed 2 tests in one commit. Please separate them to 2 different commits.

**Review by Maria7725 — CHANGES_REQUESTED** [review]
